metadata_backend/api/operators.py

```diff
@@ -57,10 +58,34 @@ def create_metadata_object(self, type: str,
         """
         accession_id = self._generate_accession_id()
         self._handle_data_and_add_to_db(type, data, accession_id)
-        LOG.info(f"Inserting file to database succeeded: {type}, "
-                 f"{self.content_type}")
+        LOG.info(f"""Inserting file to database succeeded: {type}
```

Comment:
the same here did not notice the `type` - maybe `schema_type`

Reply:
yeah that's a fair point. I think type is also in other modules, so I'll go through the whole project to fix those.